lint/action.yaml

```diff
@@ -30,6 +30,10 @@ inputs:
   npm-token:
     required: false
     description: The Node Package Manager (npm) authentication token. This token is used to authenticate against the NPM registry.
+  internal-dep-prefix:
```

Comment:
One more thought. What if we want to support multiple prefixes?

Reply:
We could potentially support a comma separated list of values 💡

Reply:
https://github.com/open-turo/actions-node/pull/22/commits/67b460b6c38a7b631384c6b002fa58a013ac285a